### ISSUE TITLE
[Feature] ramp-up of parallel tests

### ISSUE THREAD

**Issue by mstepin** [microsoft/playwright#16942]
**context:**
Playwright does well with parallelisation, and the doc describes ways in which a user can control order, serialisation, number of worker threads etc.
It also does well with detecting, based on system resources, how many workers to use if none are specified. We find the number of workers auto-defined to be quite good and accurate and want to continue using this feature.
Playwright, once it establishes the number of workers, then fires all threads off at once in parallel. Most of our tests do initial data setups for the test at the beginning of the test, and when all tests do the same or very similar actions all at once, it overwhelms the microservice that governs this. Given this only happens at the start because of this simultaneous execution, it would be a shame to reduce the number of workers (and therefore the overall test execution time in CI/CD) simply because the first 30 seconds are overkill.

**feature:**
It would be extremely helpful for us, in CI/CD to be able to define a ramp-up (in seconds or milliseconds) of the workers threads. Eg. when first run a `--ramp-up 5` option would start each thread 5 seconds apart. This would only be at first assignment of threads. Any subsequent reassignment of workers would not require a ramp up.

**Comment by pavelfeldman** [comment]
It is an interesting and a valid request, but it is unlikely that we prioritize it. I'm keeping it open to collect the upvotes.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
